input/pagecontent/reference.md

```diff
@@ -2889,6 +2892,38 @@ A path rule has no impact on the element it refers to. The only purpose of the p
     * family MS
   ```
 
+{%include tu-div.html%}
+* Include optional fixed values of a path in an Instance:
+
+  Given a profile where name.family is optional and has a fixed value, such as:
+
+  ```
+  * name.family = "Smith"
+  ```
+
+  an Instance of that profile can include the fixed value "Smith" by including the following path rule:
+
+  ```
+  * name.family
+  ```
+
```

Comment:
Do you think it's worth adding the other flavor too? Like:

> This also works when an optional element has child elements with required fixed values. In this case, a path rule to the optional element triggers the fixed values of its required child elements. For example, given a profile where name is optional and name.family is required with a fixed value, such as:
>
> ```
> * name.family 1..1
> * name.family = "Smith"
> ```
>
> an Instance of that profile can include a name with the fixed value "Smith" by including the following path rule:
>
>  ```
>  * name
>  ```

Do you think this is helpful (good)? Or unnecessary (neutral)? Or confusing (bad)?

Reply:
Oh right, I forgot about that piece. Sorry! I think it is helpful (good). I'll write it out and let you know if I change my mind.

Reply:
I added something similar to what you wrote in [8dca2ef](https://github.com/HL7/fhir-shorthand/pull/152/commits/8dca2efaab673c6f7af3c6f6c5f36737bb883ab8). I did reword it a little to try to match the writing style in the section, but if you feel that it is less clear, I can go back to what you suggested.

I also swapped the order of the use case for including optional fixed values and setting the order of slices so that slices came first, since it felt like that gave a little more emphasis to the slice order use case and that is the one that likely will be more common.

Reply:
Looks good to me!